ruby/rspec/chrome_example_spec.rb

```diff
@@ -24,9 +24,14 @@
   before(:each) do |test|
     caps = {
         browser_name: 'chrome',
-        platform_version: 'Windows 10',
+        # platform_version now becomes platform_name with the W3C bridge.
+        # For more details see: https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/Remote/W3C/Capabilities
+        platform_name: 'windows 10',
         browser_version: '61.0',
```

Comment:
It looks like Chrome is up to version 71.0, which works fine according to some tests I've run.

Reply:
My logic was to use the minimum version in order to indicate that W3C works from that version up. For the next push I'll update the browsers to their latest versions if that's what we decide.